layouts/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="{{ .Site.LanguageCode }}">
 {{- partial "head.html" . -}}
 
-<body data-theme="{{ .Site.Params.defaultColor }}" class="notransition">
+<body data-theme="{{ .Site.Params.defaultColor | default "auto" }}" class="notransition">
```

Comment:
@kusyka911, I like what you have done here: setting the default color mode to auto.